src/components/Sidebar/Sidebar.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Sidebar from './Sidebar';
+
+describe('<Sidebar />', () => {
+  it('should render and match the snapshot when closed', () => {
+    const props = {
+      open: false,
+      onClose: jest.fn()
+    };
+    const actual = mount(<Sidebar {...props} />);
```

Comment:
If I'm not mistaken, it's more performant to use `create()` for snapshot testing. 

Edit: I see that you're using `create()` for other components, so I guess this was just an oversight?